Backend/PersonDetection.Backend.Application/Common/Models/Requests/Login/LoginRequestValidator.cs

```diff
@@ -0,0 +1,21 @@
+using FluentValidation;
+using Microsoft.Extensions.Options;
+using PersonDetection.Backend.Infrastructure.Common.Options;
+
+namespace PersonDetection.Backend.Application.Common.Models.Requests.Login;
+
+public class LoginRequestValidator : AbstractValidator<LoginRequest>
+{
+    public LoginRequestValidator(IOptions<IdentityModelOptions> options)
+    {
+        RuleFor(loginRequest => loginRequest.UserName)
+            .NotEmpty().WithMessage("Username is required")
```

Comment:
For such messages, you can create a separate class that will store string constants. This will remove the potential risk of writing errors and give some flexibility